sphinx/sphinx.gradle

```diff
@@ -43,7 +43,7 @@ def sphinxDockerfile = tasks.register('sphinxDockerfile', Dockerfile) {
     runCommand '''set -eux; \\
                   mkdir /tmp/devroot; \\
                   mkdir /tmp/workspace; \\
-                  python -m pip install sphinx==4.0.3 sphinx-autodoc-typehints==1.12.0 pyarrow==6.0.1 protobuf==3.17.3 grpcio==1.39.0 bitstring==3.1.9 /wheel/*.whl breathe==4.31.0 furo==2021.9.8
+                  python -m pip install sphinx==4.0.3 sphinx-autodoc-typehints==1.12.0 pyarrow==6.0.1 protobuf==3.20.1 grpcio==1.44.0 bitstring==3.1.9 /wheel/*.whl breathe==4.31.0 furo==2021.9.8
```

Comment:
pyarrow too here i suspect, though it probably matters less on this one (should pick up the change transitively from the pyclient wheel)

Reply:
Done and done.